.appveyor.yml

```diff
@@ -2,17 +2,27 @@ version: 0.8.{build}
 environment:
   matrix:
     # array of all environments used to test builds
+    - GENERATOR: NMake Makefiles
+      CFG: Debug
+      VS_VERSION: 12.0
     - GENERATOR: Visual Studio 14 2015
+      VS_VERSION: 14.0
       CFG: Debug
     - GENERATOR: Visual Studio 12 2013
+      VS_VERSION: 12.0
       CFG: Debug
     - GENERATOR: Visual Studio 14 2015 Win64
       CFG: Debug
+      VS_VERSION: 14.0
     - GENERATOR: Visual Studio 12 2013 Win64
       CFG: Debug
+      VS_VERSION: 12.0
+install:
+  - gem install asciidoctor
```

Comment:
Its tiny so far as I’ve seen.  There are ways to cache the results, but
that adds a lot of complexity so I’m going to avoid that until we see this
is a problem.

On Mon, Apr 25, 2016 at 10:44 AM, Jack R. Dunaway <notifications@github.com>
wrote:

> In .appveyor.yml
> <https://github.com/nanomsg/nanomsg/pull/665#discussion_r60955592>:
>
> >      - GENERATOR: Visual Studio 12 2013 Win64
> >        CFG: Debug
> > +      VS_VERSION: 12.0
> > +install:
> > +  - gem install asciidoctor
>
> How much time does this add to the build? Probably not much, but it's
> worth monitoring.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nanomsg/nanomsg/pull/665/files/cb6ea68a431d182e12a7700a86e509a1fc74df9a#r60955592>
>